src/pixi.inspector.js

```diff
@@ -314,23 +321,26 @@
 			return false;
 		},
 		node: function (node) {
-			var inspector = node._inspector;
-			if (!inspector) {
-				inspector = {
-					id: this.generateId(),
-					collapsed: true,
-					type: this.detectType(node)
-				};
-				node._inspector = inspector;
-			}
+			var inspector = node._inspector || {};
+
+			var defaultTo = function (obj, prop, value) {
+				if (obj[prop] === undefined) { obj[prop] = value; }
+			};
+
+			defaultTo(inspector, 'id', this.generateId());
+			defaultTo(inspector, 'collapsed', true);
+			defaultTo(inspector, 'type', this.detectType(node));
+
+			node._inspector = inspector;
```

Comment:
This change is intended to make `node._inspector` more robust, the previous code failed given a partial object. I see `_inspector` as a convenient mechanism for annotating objects with inspector-related metadata, so I'd like to see it hardened for use in user code.